lib/shared/entities/category.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class Category {
+  final int id;
+  final String name;
+  final int transactionTypeId;
```

Comment:
Ao invés de usar o id, devemos ter o objeto TransactionType aqui
```suggestion
  final TransactionType transactionType;
```